ThunderCloud/QuizQuestionContainerViewController.swift

```diff
@@ -335,18 +335,39 @@ open class QuizQuestionContainerViewController: AccessibilityRefreshingViewContr
         
         progressView.progress = Float(quiz.currentIndex) / Float(questions.count)
         progressView.set(minY: progressView.frame.minY + 10)
-        progressView.transform = progressView.transform.concatenating(CGAffineTransform(scaleX: 1.0, y: 3.0))
+        
+        let progressViewHeight: CGFloat = 6
+        // This is according to Stack Overflow the best way to give `UIProgressView` a custom height.
+        // we're not hard-coding the denominator here, as in iOS 14 Apple changed the standard height.
+        // frame may be set to 22 above, but something in `UIProgressView`'s init method, manually sets
+        // it to the correct system default value!
+        let yTransform = progressViewHeight/progressView.bounds.height
+        progressView.transform = progressView.transform.concatenating(CGAffineTransform(scaleX: 1.0, y: yTransform))
```

Comment:
Not ideal but if it's all that seems to work!

Reply:
Yeah it's a bit shitty :(